### PR TITLE
E2E tests for adding visual component to Playlist 

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
E2E tests for adding a component to a Playlist other than Embedded Presentation

## Motivation and Context
Playlist UI epic

## How Has This Been Tested?
Tests pass
[stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: test/e2e/template-editor/cases/template-editor-add.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ var TemplateAddScenarios = function() {
       });
 
       it('should delete the Presentation', function () {
-        browser.sleep(500);
+        browser.sleep(10000); //TODO: prevent extra Save calls from happening
```
Comment (alex-deaconu) [inline test/e2e/template-editor/cases/template-editor-add.js:87]: @ezequielc curious why not use the `waitForAutosave()` utility function? Thanks.

Comment (ezequielc) [reply test/e2e/template-editor/cases/template-editor-add.js:87]: @alex-deaconu I tried using it but it didn't solve the problem. My assumption is that an autosave timeout is in progress while the presentation gets deleted and does not resolve properly, causing protractor to throw a script timeout.

Ideally the presentation should not flag autosave when just created, so I think we could queue a task to fix this.
But for now, given we want to remove autosave feature soon, I propose we proceed with this workaround. Thoughts? Thanks!

Comment (alex-deaconu) [reply test/e2e/template-editor/cases/template-editor-add.js:87]: @ezequielc we talked, seems the template used has some deleted images and is trying to update the metadata from them. We will use a different template for Add/Remove e2e tests. Thanks.

Comment (ezequielc) [reply test/e2e/template-editor/cases/template-editor-add.js]: @alex-deaconu fyi, updated as per discussion. Thanks for the help!

**Review by alex-deaconu — APPROVED** [review]
👍

**Pull request merged**